sparkle/src/components/AssistantPreview.tsx

```diff
@@ -79,10 +79,7 @@ const ItemVariantContent = ({
   return (
     <>
       <div className="s-flex s-items-center s-gap-2">
-        <Avatar
-          visual={<img src={pictureUrl} alt={`Avatar of ${title}`} />}
-          size="sm"
-        />
+        <Avatar name={`Avatar of ${title}`} visual={pictureUrl} size="sm" />
```

Comment:
The `<img>` tag is not needed since `Avatar` already bundles this behavior if `visual` is a string.

Reply:
Ack; did you double check that the behaviour was the same with the `Avatar of ${title}` string as `name` prop rather than as alt text of an img?

Reply:
Yep, if `visual` is of type `string`, the `name` props is used as an alt on the `img` tag:
https://github.com/dust-tt/dust/blob/d5b08f83625acc75bb68f5f1e1a2849db114a1d8/sparkle/src/components/Avatar.tsx#L154-L162

Reply:
:+1: and `name` doesn't do other things that alt didn't and that might be unwanted?

Reply:
Nope no side effects.